demo/js/MIDI/Player.js

```diff
@@ -0,0 +1,284 @@
+/*
```

Comment:
3rd party code without reference? Either use a git submodule or contribute to the project to release a npm package